### DISCUSSION TITLE
Modem defaults to IPv4 mode with 464CLAT enabled.

### DISCUSSION THREAD

**Discussion by ghost** [iamromulan/cellular-modem-wiki#82] [Help - Quectel] [answered]
I want to know if it is possible to disable CLAT and use it in pure IPv6 mode so that the router with 464XLAT capability can do the IPv4?

Right now, I am unable to get DHCP IPv6 via SLAAC or RA to any of my LAN devices.

**Comment by iamromulan** [comment]
I will look into this and let you know 👍

**Comment by ghost** [comment]
Hello. To add to this problem, I managed to get m2. to RJ45 module to work with RM551E-GL by setting these 2 AT commands
`AT+QCFG="usbnet",1
AT+QCFG="pcie/mode",1`
However, only 1st Device or Router gets IPv6 address. All LAN devices cannot request IPv6 via SLAAC or DHCPv6 RA. Since I am using Ubiquiti UXG-MAX, there is no option of IPv6 Passthrough. Is it some setting on the RM551E-GL that needs to be changed or it is purely Ubiquiti problem?

https://i.imgur.com/OyTrdi2.jpeg

**Comment by iamromulan** [comment]
> Hello. To add to this problem, I managed to get m2. to RJ45 module to work with RM551E-GL by setting these 2 AT commands
> `AT+QCFG="usbnet",1
> AT+QCFG="pcie/mode",1`
> However, only 1st Device or Router gets IPv6 address. All LAN devices cannot request IPv6 via SLAAC or DHCPv6 RA. Since I am using Ubiquiti UXG-MAX, there is no option of IPv6 Passthrough. Is it some setting on the RM551E-GL that needs to be changed or it is purely Ubiquiti problem?
> 
> https://i.imgur.com/OyTrdi2.jpeg

I'm still trying to understand how they have IPPT set up internally.

It seems to me they still have some work to do on firmware for ECM/RNDIS/QMAP.

In Luci go to interfaces and inspect the lan_bind6 interface. That's a good place to start.

**Comment by iamromulan** [comment]
Some time has passed here. Does this still happen on December Firmware?

**Reply by ghost** [reply]
Yes

**Comment by clndwhr** [comment, marked as answer]
Ubiquiti still doesn't have full 464XLAT support on their network line. You'll get the IPV6 delegated address to your gateway and you can get some systems to connect under SLAAC and DHCPv6, i had issues with IoT and some older Android based devices getting network connection. None of my devices when testing this, even on the newer Dec and March firmware would function solely on IPV6 through my Ubiquiti UDM or UCG devices, regardless of SLAAC or DHCPv6.  It would seem the Unifi system has issues with handling and passing the IPV6 through to the devices. The best IPV6 support with the 551E in IPPT mode with Unifi Network was using SLAAC (prefix 64) on WAN with IPv4/v6(SLAAC/64) on LAN side but this also broke half of my IoT devices (1 of my 2 roborock vacs, a couple gen2 chromecasts, my August smart lock, and a few smart outlets). I found no benefits or gains running IPPT to my Unifi Network devices over letting the RM551E handle the 464 functionality, and that includes testing with gaming, VOIP, video meetings, and VPN (I WFH on the TMHI connection), latency in IPPT mode or otherwise usually lives around 25ms for me.

My total main line setup is an RM551 Primary WAN, RM520 Secondary(Load Balanced wtih 1%, connected to assist with development/integration effort). The RM551 was a 4x4 MIMO Directional Waveform Pro connected with tower at 4.8 miles away through some trees a couple barns, a grain silo and over the horizon.

The RM551 is running @iamromulan custom firmware with the 4CA fix implemented, the signal statistics on the PCC N41 band at 100Mhz width are -74RSRP, -10RSRQ, ~25SINR. I get an additional N41 SCC at 90Mhz width, and generally an N71 at 15Mhz or 2 N25 (1 at 20Mhz and 1 at 5Mhz), All SCC bands have similar characteristics with RSRP at ~-85, RSRQ@-15, SINR#~11. Best connection is Dual N41, Dual N25 at avg download 2.1Gbps and avg upload 150Mbps. Antenna signal quality sits about 96% and Signal Strength floats around 93%.  I occasionally test NSA mode and signal strengths are similar for the LTE bands but the speeds are horrible to say the nicest.
